homeassistant/components/litterrobot/sensor.py

```diff
@@ -36,19 +36,39 @@ def unit_of_measurement(self):
     @property
     def icon(self):
         """Return the icon to use in the frontend, if any."""
-        if self.robot.waste_drawer_gauge <= 10:
-            return "mdi:gauge-empty"
-        if self.robot.waste_drawer_gauge < 50:
-            return "mdi:gauge-low"
-        if self.robot.waste_drawer_gauge <= 90:
-            return "mdi:gauge"
-        return "mdi:gauge-full"
+        return icon_for_gauge_level(self.state, 10)
+
+
+class LitterRobotSleepTimeSensor(LitterRobotPropertySensor, Entity):
+    """Litter-Robot sleep time sensors."""
 
     @property
-    def device_state_attributes(self):
-        """Return device specific state attributes."""
-        return {
-            "cycle_count": self.robot.cycle_count,
-            "cycle_capacity": self.robot.cycle_capacity,
-            "cycles_after_drawer_full": self.robot.cycles_after_drawer_full,
-        }
+    def state(self):
+        """Return the state."""
+        if self.robot.sleep_mode_active:
+            return super().state
```

Comment:
Is this an an isoformatted datetime?
```suggestion
            return super().state
        return None
```